### ISSUE TITLE
[FEEDBACK] Issues related to the controls visualization

### ISSUE THREAD

**Issue by Hannes-Keunen** [opensearch-project/OpenSearch-Dashboards#4166]
I am currently trying to create some dashboards that are easy to use even for non-technical users. However, I ran into some issues related to the controls visualization that might be confusing to some users. Because the dashboards will be used by non-technical users, I want to avoid having to use the dashboard filters as much as possible, in favor of the controls panel. I replicated some scenarios using the flights demo dashboard so it's easier to explain.

**1. The controls visualization is not affected when a dashboard filter is added for the same field.**
This can be replicated in the flights dashboard by filtering on OriginDestinationName in the flights log table on the bottom.
![image](https://github.com/opensearch-project/OpenSearch-Dashboards/assets/38217264/f9e12e13-9725-4906-a12c-7507129f3756)

**2. Similar to 1, if a dashboard filter filters out values from the control for a different field, those are still shown in the dropdown for the control**
For example, if I add a dashboard filter for Carrier: Logstash Airways, the dropdown for origin city still contains an entry for Abu Dhabi, even though Logstash Airways has no flights from Abu Dhabi. This might be confusing for some users.
![image](https://github.com/opensearch-project/OpenSearch-Dashboards/assets/38217264/d22a3c97-bf26-4144-92e7-5d6c72a6463e)

**3. There is no simple way to exclude values via the controls visualization**
The current workaround is to select a value in the control, then click the newly added dashboard filter and select 'exclude results'. The dashboard is then filtered correctly, but the control still shows the selected value as if it were included, which is very confusing.
![image](https://github.com/opensearch-project/OpenSearch-Dashboards/assets/38217264/e773852e-4615-4347-b7c1-5c1983e04f39)

Ideally the controls panel itself would have an option to exclude items. Maybe that could be achieved by adding small '+' and '-' icons next to each item in the dropdown when you hover it, just like in the saved search panel?

So in short, the main problem is that the control panel does not seem to be affected by the dashboard filters. Is that intended for some reason, or can this be fixed? Also, if there are better workarounds for any of those issues, please let me know!

**Comment by ananzh** [comment]
@KrooshalUX  could you help us to take a look and provide some UX input?

**Comment by joshuarrrr** [comment]
Thanks @Hannes-Keunen, this is very useful feedback. We inherited this experimental feature, and it definitely has room for improvement. I think the question is the long-term product and UX plans for filtering use cases.

**Comment by kavilla** [comment]
To avoid duplication as this seems to be a general issue, will drop another UX issue with the controls visualization.

Right now the controls visualization autocomplete will output `Terms list might be incomplete because the request is taking too long. Adjust the autocomplete settings in opensearch_dashboards.yml for complete results`. However, depending on the environment, non-admin users might not be able to to modify the config file.

@opensearch-project/opensearch-ux do you think the wording could be something on the lines of the other parts of the project? (Since we do not have the i18n plugin distributed). Something along the lines of `Terms list might be incomplete because the request is taking too long. Contact your system administrator to review your autocomplete settings for complete results`?

**Comment by kgcreative** [comment]
@kavilla - yes, this makes sense to me. cc: @vagimeli for language review, and making sure that we have documentation for this setting.
